### ISSUE TITLE
To make website mobile friendly for users

### ISSUE THREAD

**Issue by girija0707** [publiclab/infragram#239]
The website at the moment does not fit the screen size perfectly
![Screenshot_2022-03-27-16-08-58-12_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/63583646/160278043-12ec5dee-7080-4efa-a9fd-e7013de7460b.jpg)
would love working on website to make it fit for any screen size

**Comment by girija0707** [comment]
Would like to know can this issue be a part of proposal @TildaDares

**Comment by TildaDares** [comment]
Hi @girija0707, yes this can be a part of your proposal.

**Comment by girija0707** [comment]
Ok , glad to know that will be working on it

**Comment by girija0707** [comment]
Hey @TildaDares can you help me with this doubt, should I start working on resolving this issue or look for few more issues to add them in the proposal and finally start working on resolving them  if and when  the proposal is accepted

**Comment by girija0707** [comment]
Hello @TildaDares would love to know should I start working on resolving this issue now or explore and find more issues to write them in proposal and start working on them after the proposal is accepted

**Comment by steph237** [comment]
Greetings @TildaDares I will like to know if i can work on this issue for outreachy application

**Comment by steph237** [comment]
@TildaDares  greetings I cloned the project but when I use the command http-serve , this is what I have on my browser. I am wondering if I missed something. will be waiting on your reply. Thank you
![30 03 2022_09 21 47_REC](https://user-images.githubusercontent.com/43364084/160786507-cad69ac3-1218-45c0-a621-82526aecd1d6.png)

**Comment by steph237** [comment]
Yes @TildaDares I have restarted the server multiple times but i have the same output

**Comment by steph237** [comment]
hello Tilda i resolved the issue it was with the link to my directories
![30 03 2022_15 06 05_REC](https://user-images.githubusercontent.com/43364084/160854854-646351d9-92c7-41cc-9a2c-a7251ae6b593.png)
